addon/components/polaris-tag.js

```diff
@@ -1,10 +1,13 @@
 import Component from '@ember/component';
 import layout from '../templates/components/polaris-tag';
+import { computed } from '@ember/object';
```

Comment:
Should this be before the `layout` import?

Reply:
can do